packages/boutique/backend/src/cache/one-click.ts

```diff
@@ -8,4 +8,7 @@ export type OneClickCacheData = {
   continueToken: string
 }
 
-export type OneClickCache = InMemoryCache<OneClickCacheData>
+export type OneClickCache = Cache<OneClickCacheData>
```

Comment:
why the cache type was changed?

Reply:
I thought it would be better for all parts of the backends to use the same cache type at this case `OneClickCach` structure does not depend on `in-memory`  cache type so I decided to change it to the Redis type and it caused we don't worry about its expiration anymore so I cleared its expiration process in `app.ts`

Reply:
I think we should keep the same implementation. @raducristianpopa any thoughts?

Reply:
I agree with @dragosp1011. Using Redis for `OneClickCache` is not really necessary and we should stick with the in memory one.